clang/lib/Driver/ToolChains/Cuda.cpp

```diff
@@ -682,6 +688,9 @@ void NVPTX::getNVPTXTargetFeatures(const Driver &D, const llvm::Triple &Triple,
   case CudaVersion::CUDA_##CUDA_VER:                                           \
     PtxFeature = "+ptx" #PTX_VER;                                              \
     break;
+    CASE_CUDA_VERSION(129, 87);
```

Comment:
Looks like we forgot about adding PTX87 to the builting in the clang/include/clang/Basic/BuiltinsNVPTX.def

Right now `cc1` ends up running with `-target-features +ptx87`, but the target builtins only enumerate up to ptx86, so the compilation fails with 
```
In file included from trunk/include/__clang_cuda_runtime_wrapper.h:473:
trunk/include/__clang_cuda_intrinsics.h:179:40: error: '__nvvm_shfl_sync_down_i32' needs target feature ptx60|ptx61|ptx62|ptx63|ptx64|ptx65|ptx70|ptx71|ptx72|ptx73|ptx74|ptx75|ptx76|ptx77|ptx78|ptx80|ptx81|ptx82|ptx83|ptx84|ptx85|ptx86
  179 | __MAKE_SYNC_SHUFFLES(__shfl_down_sync, __nvvm_shfl_sync_down_i32,
      |                                        ^
```